src/corehost/cli/deps_format.cpp

```diff
@@ -68,7 +63,7 @@ void deps_json_t::reconcile_libraries_with_targets(
                 size_t pos = library.first.find(_X("/"));
                 entry.library_name = library.first.substr(0, pos);
                 entry.library_version = library.first.substr(pos + 1);
-                entry.library_type = _X("package");
+                entry.library_type = pal::to_lower(library.second.at(_X("type")).as_string());
```

Comment:
@anurse, can you or @brthor help answer this to assess impact?

@gkhanna79, the host doesn't have to do a full probe unless it is type package. That's an optimization.